controllers/apply/lib/field-types/field.js

```diff
@@ -13,19 +17,14 @@ class Field {
         if (props.locale) {
             this.locale = props.locale;
         } else {
-            throw new Error(`Must provide locale for ${props.name}`);
+            throw new Error('Must provide locale');
         }
 
-        // Used to switch on non-class type fields
-        // @TODO: Remove this after fully migrating to field types
-        this._isClass = true;
-
         const label = props.label ? props.label : this.defaultLabel();
         if (label) {
             this.label = label;
         } else {
-            // @TODO: Re-enable once welsh translations have been added
-            // throw new Error('Must provide label');
+            throw new Error('Must provide label');
```

Comment:
This will throw an error when we enable Welsh for the standard form, but that's a good thing as it will show where we're missing translations for labels.